File: DFRobot_RGBLCD/cpp/SetColor/SetColor.cpp
#include "../DFRobot_RGBLCD.h"
#include <iostream>
#include <string>

using namespace std;

int main(){
  int r,g,b;

  DFRobot_RGBLCD lcd(16, 2);
  // initialize
  lcd.init();
  // Print a message to the LCD.
  lcd.print("set cllor");
  
  while(1){
    cout << "r:";
    cin >> r;
    cout << "g:";
    cin >> g;
    cout << "b:";
    cin >> b; 
          
    lcd.setRGB(r, g, b);
  
    cout << "r = " << r 
         << "g = " << g
         << "b = " << b
         << endl;
  }
  return 0;
}

File: DFRobot_RGBLCD/python3/SetColor/SetColor.py
import sys
sys.path.append('../')
import rgb1602
import time

lcd=rgb1602.RGB1602(16,2)                               #create LCD object,specify col and row

# Print a message to the LCD.
lcd.print("set color")
print("set color")
while True:
  print("r:",end = '')
  r = int(input())
  print("g:",end = '')
  g = int(input())
  print("b:",end = '')
  b = int(input())    
        
  lcd.setRGB(r, g, b)

  print("get data:r = %s,g = %s,b = %s" %(r,g,b))


